lib/TokenManager.js

```diff
@@ -0,0 +1,84 @@
+var util = require('./util');
+var AuthSdkError = require('./errors/AuthSdkError');
+var config = require('./config.json');
+var cookies = require('./cookies');
+
+function tokenStorageBuilder(storage) {
```

Comment:
ok. I'll go ahead and do that. The separation makes sense anyway, because we have `clientBuilder.js`.